docs/maps/connect-to-ems.asciidoc

```diff
@@ -35,3 +35,188 @@ To disable EMS, change your <<settings, kibana.yml>> file.
+|===
+
+[float]
+[[elastic-maps-server-bind-mount-config]]
+===== Bind-mounted configuration
+
+One way to configure {hosted-ems} is to provide `elastic-maps-server.yml` via bind-mounting. With +docker-compose+, the bind-mount can be specified like this:
+
+["source","yaml",subs="attributes"]
+--------------------------------------------
+version: '2'
+services:
+  {hosted-ems}:
+    image: {ems-docker-image}
+    volumes:
+      - ./elastic-maps-server.yml:/usr/src/app/config/elastic-maps-server.yml
+--------------------------------------------
+
+[float]
+[[elastic-maps-server-envvar-config]]
+===== Environment variable configuration
+All configuration settings can be overridden by environment variables that are named with all uppercase letters and by replacing YAML periods with underscores. For example `elasticsearch.ssl.certificate` could be overridden by the environment variable `ELASTICSEARCH_SSL_CERTIFICATE`. Boolean variables must use the `true` or `false` strings.
+
+WARNING: All information that you include in environment variables is visible through the `ps` command, including sensitive information.
+
+These variables can be set with +docker-compose+ like this:
+
+["source","yaml",subs="attributes"]
+----------------------------------------------------------
+version: '2'
+services:
+  {hosted-ems}:
+    image: {ems-docker-image}
+    environment:
+      ELASTICSEARCH_HOST: http://elasticsearch.example.org
+      ELASTICSEARCH_USERNAME: 'ems'
+      ELASTICSEARCH_PASSWORD: 'changeme'
+----------------------------------------------------------
+
+[float]
+[[elastic-maps-server-data]]
+==== Data
+
+{hosted-ems} hosts vector layer boundaries and vector tile basemaps for the entire planet. Boundaries include world countries, global administrative regions, and specific country regions. A minimal basemap is provided with {hosted-ems}. This can be used for testing environments but won't be functional for normal operations. The full basemap (around 90GB file) needs to be mounted on the Docker container for {hosted-ems} to run normally.
+
+TIP: The available basemaps and boundaries can be explored from the `/maps` endpoint in a web page that is your self-managed equivalent to https://maps.elastic.co
+
+
+[float]
+[[elastic-maps-server-kibana]]
+==== Kibana configuration
+
+With {hosted-ems} running, add the `map.emsUrl` configuration key in your <<settings, kibana.yml>> file pointing to the root of the service. This setting will point {kib} to request EMS basemaps and boundaries from {hosted-ems}.
+
```

Comment:
consider adding example:

```
map.emsUrl: https://localhost:8080
```

matching the example above when setting up Elastic Maps Server